### DISCUSSION TITLE
Linux Usage

### DISCUSSION THREAD

**Discussion by kubilus1** [kubilus1/autoortho#5] [General]
Discuss using Autoortho on Linux

**Comment by h-a-l-t-e-r** [comment]
No problem so far. Started with version 0.4.0 on XP12 on Ubuntu 22.04 LTS

**Reply by h-a-l-t-e-r** [reply]
After upgrade to 0.4.1 i started to have multiples CTD. In 4 flights in a row, i disabled autoortho in my fifth flight and have no problem. I will try again with version 0.4.1 increasing the max wait time in the config.

**Comment by h-a-l-t-e-r** [comment]
I have to downgrade from 0.4.1 to 0.4.0-rc0 after several CTD in different airports, different aircrafts.

**Comment by hawajka76** [comment]
Installed recently and it is working good. The main issue which I have is with the naming convention of some folders which use an asterisk. I keep my  Custom Scenery folder on a NTFS formated drive, which does not support such naming conventions. Is there a way to update that app to change this? It would help a lot!
